### PR TITLE
Deploy Budget App 

### PR DESCRIPTION
**Pull request by RichardChileya**
In this Pull request I:
- [ ] check the code is working 
- [ ] Update the read me 
- [ ] fix linters 
- [ ] Add the project to render
- [ ] Deploy the project

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,218 @@
...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo](https://budget-app-3c3n.onrender.com)
+- [Video](https://drive.google.com/file/d/19fGLUPm0SxLqDOKr1xkhpyk1G5hcQRDh/view?usp=sharing)
+
```
Comment (uwadonat) [inline README.md]: Thank you for your effort to make a clear video. Please, let's add other project requirements and make it more attractive for viewers or recruiters.
- [x] The video should contain a highlight about a piece of the code the student finds interesting or is proud of making. this requires moving into your VS code and show a specific part of code. Kindly work on this.

**Review by uwadonat — CHANGES_REQUESTED** [review]
## Hi @RichardChileya, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- Your commit messages are very descriptive ✔️
- Amazing design ✔️
- The deployed application works fine ✔️
- The project was tested ✔️
- Correct workflow was used ✔️
- No linter errors ✔️
almost

## Required Changes ♻️
_Check the comments under the review. _

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100%.
Please, remember to tag me @uwadonat in your question so I can receive the notification.

------
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (RichardChileya) [reply README.md]: Thank you for your suggestions, i have recorded a new video highlighting my project (https://drive.google.com/file/d/16oDY-3v6GjoWdDSYWB2Khpmdg_u8z9xG/view?usp=sharing) and i have updated the link in my readme file

**Comment by arslanbisharat** [comment]
Hi @RichardChileya,
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- The Pull Request has a proper title and description. ✔️ 
- You have added a descriptive Readme file. Good Job. ✔️ 
- There is no linter error is present. ✔️ 
- Project has met all the technical aspects you needed for this project. Keep rocking! ✔️ 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**